src/Response.php

```diff
@@ -194,9 +205,8 @@ public function getId()
      * @param  string $version
      * @return self
      */
-    public function setVersion($version)
+    public function setVersion(?string $version) : self
     {
-        $version = (string) $version;
         if ('2.0' == $version) {
```

Comment:
strict comparison here

(also no yoda, but this can be addressed later).

Reply:
Will push strict comparison shortly.